packages/web-components/src/components/ic-select/ic-select.e2e.ts

```diff
@@ -1910,6 +2106,46 @@ describe("ic-select", () => {
       </ic-input-container>`);
     });
 
+    it("should render a native select when searchable is applied on mobile", async () => {
+      await page.setContent(getTestSearchableSelect(options));
+      await page.waitForChanges();
+
+      const select = await page.find("ic-select");
+
+      expect(select.shadowRoot)
```

Comment:
could this be moved into the snapshot file? - there is a separate one for e2e tests